packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-list.jsx

```diff
@@ -9,14 +9,18 @@ const List = ({ handleInfoClick, handleSelect, list, name, value }) =>
     list.map((contract_category, key) => {
         const contract_types = contract_category.contract_types?.filter(contract_type => {
             const base_contract_type = /^(.*)_equal$/.exec(contract_type.value)?.[1];
+            const is_hidden = contract_type.value === 'turbosshort';
 
+            if (is_hidden) {
+                return contract_category.contract_types.some(c => c.value === is_hidden);
```

Comment:
is c.value boolean though? if we wanted it to be boolean maybe

!!c.value ?

Reply:
good point ) this line is unnecessary at all because it will false all the time (string is compared strictly with boolean). we'll fix it👍🏻